hummingbot/strategy/cross_exchange_market_making/cross_exchange_market_making.pyx

```diff
@@ -867,14 +867,7 @@ cdef class CrossExchangeMarketMakingStrategy(StrategyBase):
                 )
                 price_above_bid = (ceil(top_bid_price / price_quantum) + 1) * price_quantum
 
-            try:
-                taker_price = taker_market.c_get_vwap_for_volume(taker_trading_pair, False, size).result_price
-            except ZeroDivisionError:
-                return s_decimal_nan
-
-            # If quote assets are not same, convert them from taker's quote asset to maker's quote asset
-            if market_pair.maker.quote_asset != market_pair.taker.quote_asset:
-                taker_price *= self.market_conversion_rate()
+            taker_price = self.c_calculate_effective_hedging_price(market_pair, is_bid, size)
```

Comment:
Are the changes in XEMM module related to config migration?

Reply:
No, this is something that I noticed while investigating an issue with XEMM. The code I've removed is a duplicate of `c_calculate_effective_hedging_price`.